src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
> Should I have used the computed value of 0 instead of 0 << 16 here for example.

Ya, I was wondering the same thing. I think `0 << 16` is sufficiently enough of a head scratcher to defeat any case for the argument in its favor. I think 0 here is better.

> We tend to not specify the values when they are sequential especially when the native code header 

I hadn't noticed that. In a PR I review I usually ask for the values to be explicitly added, if nothing else than to make it evident that it wasn't simply overlooked (which it often is!) when they are important to have there.